### ISSUE TITLE
Examples Don't "Re"compose

### ISSUE THREAD

**Issue by Ellisande** [garretttaco/learning-recompose#9]
One of the big benefits of recompose is the ability to re-use the compositions where they are useful. It might be worth it to have an example where two components need to share functionality and show you can easily share between them. The "loading state & triggers" is a good example of that where you could do something like `const withAsync = compose( withState('loading' ...) )(Component)` and then show you can easily add that to other components and reuse it.

Another good one might be to have two components that need to share a behavior from a handler?

**Comment by tylercollier** [comment]
I agree. I believe I tried to tell you this somewhere too, but Justin said it better.
